Bottom: 3303edb3ba0c2ed4691b0b3455c6b8c32d779432
Top:    091b97283e99bf72cde41888c3620138e16e4fd4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-26 16:13:04 +0000

WAS: Use motion hints when processing pointer motion over the drawing area

Not sure what the remaining changes here are for!


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index a272714..d3a38e4 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -748,7 +748,7 @@ ghid_show_crosshair (gboolean show)
 
   glBegin (GL_LINES);
 
-#if 1
+#if 0
   if (x_prev >= 0)
     draw_crosshair (x_prev, y_prev);
 #endif
@@ -761,7 +761,7 @@ ghid_show_crosshair (gboolean show)
 
   glBegin (GL_QUADS);
 
-#if 1
+#if 0
   if (x_prev >= 0 && draw_markers_prev)
     {
       glVertex2i (0,                  y_prev - VCD);
